phases/phase-2/challenges/add-reaction.md

```diff
@@ -0,0 +1,105 @@
+# Add a reaction to post
+:::tip Note  
+This challenge allows you to receive **30 Desmos Tokens** upon completion.  
```

Comment:
This challenge allows you to receive **30 Desmos Tokens** upon completion.  ->
You will earn **30 Desmos Tokens** upon completion of this challenge.